src/cpp/src/icontinuous_batching.cpp

```diff
@@ -133,6 +130,16 @@ ContinuousBatchingPipeline::IContinuousBatchingPipeline::generate(
     return decoded;
 }
 
+std::string ContinuousBatchingPipeline::IContinuousBatchingPipeline::add_image_tags_to_prompt(const std::string& prompt, const std::vector<ov::Tensor>& rgbs) {
+    std::stringstream prompt_with_image_tags;
```

Comment:
@Wovchena looks like here we also need to check whether tags are applied manually or not.

Do you think we need to share function to do it from InputsEmbedder?

Reply:
https://github.com/openvinotoolkit/openvino.genai/pull/1884